GitHubContents.ps1

```diff
@@ -340,6 +342,8 @@ filter Set-GitHubContent
 
         [string] $AuthorEmail,
 
+        [string] $ContentPath,
```

Comment:
Parameters should be ordered in the same position as their documentation.  In this case, this should appear right below `[string] $Content`.

Reply:
Fixed